src/Makefile.am

```diff
@@ -447,6 +447,8 @@ libbitcoin_consensus_a_SOURCES = \
   pubkey.cpp \
   pubkey.h \
   script/bitcoinconsensus.cpp \
+  script/bitfield.cpp \
+  script/bitfield.h \
   script/interpreter.cpp \
```

Comment:
add these files to .formatted-files   also add bitfeild_tests.cpp

Reply:
Good point. Will do

Reply:
done